modules/creating-manifest-file-customized-br-ex-bridge.adoc

```diff
@@ -126,24 +126,42 @@ apiVersion: machineconfiguration.openshift.io/v1
 kind: MachineConfig
 metadata:
   labels:
-    machineconfiguration.openshift.io/role: worker <1>
-  name: 10-br-ex-worker <2>
+    machineconfiguration.openshift.io/role: worker
+  name: 10-br-ex-worker <1>
 spec:
   config:
     ignition:
       version: 3.2.0
     storage:
       files:
       - contents:
-          source: data:text/plain;charset=utf-8;base64,<base64_encoded_nmstate_configuration> <3>
+          source: data:text/plain;charset=utf-8;base64,<base64_encoded_nmstate_configuration> <2>
+        mode: 0644
+        overwrite: true
+        path: /etc/nmstate/openshift/worker-0.yml <3>
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,<base64_encoded_nmstate_configuration>
+        mode: 0644
+        overwrite: true
+        path: /etc/nmstate/openshift/worker-1.yml <3>
+# ...
+----
+<1> The name of the policy.
+<2> Writes the encoded base64 information to the specified path.
+<3> For each node in your cluster, specify the hostname path to your node and the base-64 encoded Ignition configuration file data for the machine type. The `worker` role is the default role for nodes in your cluster. The `.yaml` extension does not work when specifying the hostname path for each node or all nodes in the `MachineConfig` manifest file.
```

Comment:
we should probably be explict and say "short hostname" or `hostname -s`.  

The man page is even explicit:
```
       -s, --short
              Display the short host name. This is the host name cut at the first dot.
```

Reply:
@rbbratta . Thanks. I updated the content to specify "short hostname".